moveit_setup_assistant/templates/moveit_config_pkg_template/launch/demo.launch

```diff
@@ -1,5 +1,8 @@
 <launch>
 
+  <!-- specify the planning pipeline -->
+  <arg name="pipeline" default="ompl" />
```

Comment:
This is a little bit philosophical: While the [tutorials introduce the argument `planner`](https://ros-planning.github.io/moveit_tutorials/doc/chomp_planner/chomp_planner_tutorial.html#using-chomp-with-your-robot), here the argument `pipeline` is introduced for the very same purpose.
I vote for keeping `pipeline` because this better indicates that you could have different pipeline configs for the same planner plugin, e.g. `ompl1` and `ompl2`. However, a follow-up PR should be filed for the tutorials to align them.

Reply:
I agree with keeping `pipeline`, is `planner` used in this case anywhere else but the tutorials?

Reply:
I love long variable names and would even suggest ``planning_pipeline``, but that's for another PR

@jschleicher do you mind updating the tutorials per @rhaschke ?
https://github.com/ros-planning/moveit_tutorials

Reply:
I'm not a panda expert; updating the tutorials should also re-generate the panda_moveit_config. Opened ros-planning/moveit_tutorials#300 as follow-up-issue.